how-to/workspace-platform-starter/public/manifest.fin.json

```diff
@@ -924,12 +924,70 @@
 						"consumerKey": "",
 						"orgUrl": "",
 						"preload": "https://built-on-openfin.github.io/workspace-starter/workspace/v13.0.0/integrate-with-salesforce/js/preload.js",
+						"enableLibLogging": true,
 						"iconMap": {
+							"salesforce": "https://built-on-openfin.github.io/workspace-starter/workspace/v13.0.0/integrate-with-salesforce/images/salesforce.svg",
 							"contact": "https://built-on-openfin.github.io/workspace-starter/workspace/v13.0.0/integrate-with-salesforce/images/contact.svg",
 							"account": "https://built-on-openfin.github.io/workspace-starter/workspace/v13.0.0/integrate-with-salesforce/images/account.svg",
 							"chatter": "https://built-on-openfin.github.io/workspace-starter/workspace/v13.0.0/integrate-with-salesforce/images/chatter.svg",
 							"note": "https://built-on-openfin.github.io/workspace-starter/workspace/v13.0.0/integrate-with-salesforce/images/note.svg",
-							"task": "https://built-on-openfin.github.io/workspace-starter/workspace/v13.0.0/integrate-with-salesforce/images/task.svg"
+							"task": "https://built-on-openfin.github.io/workspace-starter/workspace/v13.0.0/integrate-with-salesforce/images/task.svg",
+							"dashboard": "https://built-on-openfin.github.io/workspace-starter/workspace/v13.0.0/integrate-with-salesforce/images/dashboard.svg",
+							"phone": "https://built-on-openfin.github.io/workspace-starter/workspace/v13.0.0/integrate-with-salesforce/images/phone.svg",
+							"chat": "https://built-on-openfin.github.io/workspace-starter/workspace/v13.0.0/integrate-with-salesforce/images/chat.svg",
+							"email": "https://built-on-openfin.github.io/workspace-starter/workspace/v13.0.0/integrate-with-salesforce/images/email.svg"
+						}
+					}
+				},
+				{
+					"id": "ms365",
```

Comment:
Maybe we shouldn't include ms365 by default as this could be enabled with a microflow in 13.1.

Reply:
Removed